_data/faq.yml

```
@@ -52,3 +52,35 @@
 - question: What is Notion?
   answer: Notion is a productivity and note-taking tool which provides a variety of features to keep track of progress. We recommend using this tool during mentorship experience. Or template can help mentors and mentees be organized and to be on track. At the beginning of the cycle we will conduct a webinar on how to get the most out of Notion note-taking experience.
   index: 11
+
+- question: What are the goals and objectives of the mentorship programme? What is expected of me as a mentor?
+  answer: |
+    What are the goals and objectives of the mentorship programme? What is expected of me as a mentor? <br><br>
+
+    The goals and objectives of a mentorship programme may change in light of different needs and circumstances. However, some common goals and objectives include:<br>
+    1. Professional development: To provide mentees with guidance and support as they navigate their career and personal growth. <br>
+    2. Skill development: To help mentees develop specific skills and knowledge that will be valuable to them in their current or future roles. <br>
+    3. Career guidance: To help mentees set and achieve career goals, and to provide them with the tools and resources needed to succeed in their chosen field. <br>
+    4. Personal development: To provide mentees with guidance and support as they navigate their personal growth and well-being. <br> <br>
+    As a mentor, you will be expected to provide guidance, advice, and support to your mentee as they navigate their professional and personal development. You will be expected to share your own experiences and knowledge, and provide constructive feedback. You will also be expected to help the mentee set and achieve goals, and to introduce them to new opportunities that can help them advance in their career. <br>
+    Please, find more information in Code of Conduct for Mentors.
```

```suggestion
    Please, find more information in <a href="https://womenwhocode.github.io/london/code-of-conduct)">Code of Conduct for Mentors</a>.
```

I think that was what @nora-weisser suggested

---

you can try also: 

`<a href="/london/code-of-conduct)" target="_self">Code of Conduct for Mentors</a>.` maybe is better.